README.md

```diff
@@ -21,67 +17,226 @@ Additionally you also need to mark some of the dependencies in your dependencies
+Again: in the **shadowing only** mode no transient dependency hygiene is applied. Also the above described config options (*overrides* and *expositions*) don't take effect.
 
-A bit of additional magic happens when you use the built in profile: it actively switches on the leiningen middleware also built into the plugin. The middleware AOT compiles some clojure sources and removes dependencies marked with `^:source-deps` from the dependency list. So they won't show up in the generated pom file and so on.
+### Further config options
+
+| Option                   | Default                        | CLI or project.clj    | Description | Example |
+|--------------------------|--------------------------------|-----------------------|-------------|---------|
+| project-prefix           | mranderson{mranderson version} | CLI                   | project pecific prefix to use when shadowing            | `lein source-deps :project-prefix cider.inlined-deps` |
+| skip-javaclass-repackage | false                          | CLI                   | If true [Jar Jar Links](https://code.google.com/p/jarjar/) won't be used to repackage java classes in dependencies            | `lein source-deps :skip-javaclass-repackage true`        |
+| prefix-exclusions        | empty list                     | CLI                   | List of prefixes which should not be processed in imports            |  `lein source-deps :prefix-exclusions "[\"classlojure\"]"`  |
+| watermark                | :mranderson/inlined            | project.clj           | When processing namespaces in dependencies MrAnderson marks them with a meta so inlined namespaces can be identified. Helpful for tools like [cljdoc](https://cljdoc.org)            | `:mranderson {:watermark nil}` to switch off watermarking or provide your own keyword        |
+| shadowing-only           | false                          | CLI, project.clj      | Switch between **unresolved tree** and **shadowing only** mode | `lein source-deps :shadowing-only true` |
+| overrides                | empty list                     | project.clj           | Defines dependency overrides in **unresolved tree** mode | `:mranderson {:overrides {[mvxcvi/puget fipp] [fipp "0.6.15"]}}` |
+| expositions              | empty list                     | project.clj           | Makes transient dependencies available in the project's source files in **unresolved tree** mode | `:mranderson {:expositions [[mvxcvi/puget fipp]]}` |
+
+## Prerequisites
+
+Leiningen 2.8.3 or above. For MrAnderson to work, does not mean your project is restricted to a java or clojure version.
+
+## Is it good? Should I use it?
+
+Yes and yes. Use it if you have depedency conflicts and don't care to solve them. Or if you don't want your library's dependencies to interfere with the dependencies of your users' (leiningen plugins is a good example here). Or if you want to explore a bit more in the hellish land of dependency handling.
```

Comment:
I think this section is a bit confusing, due to lack of examples. Probably I'd move the `Why?` to the beginning of the readme. 

> Use it if you have depedency conflicts and don't care to solve them.

Here you have typo, but overall the sentence sounds odd. It's not clear what you mean by it.